### DISCUSSION TITLE
Does Serverless provides any specificationfor monitoring the running workflow instances

### DISCUSSION THREAD

**Discussion by ashyk** [serverlessworkflow/specification#765] [Q&A]
Hi Team,

We are actively exploring CNCF Serverless Spec to build a cloud based Enterprise level Workflow Solutions.
And as part of that, we are evaluating the prospect of providing the monitoring functionality for running workflow instances.
Do Serverless spec provide anything related to monitoring functionality?

**Comment by cdavernas** [comment]
Unhappily it does not for now, but that is something being talked about/drafted, as there clearly is a need to describe running and completed instances in a consistent way across runtimes.

Your observations and ideas are warmly welcome!
